### PR TITLE
Feature/extend event with fields

### PR DESCRIPTION
**Pull request by gombaniro**

### CODE REVIEW

File: src/main/java/com/tg5/service/contract/EventPayload.java [hunk 1]
```diff
@@ -12,4 +14,8 @@ public class EventPayload implements Serializable {
     private Long id;
 
     private String name;
+
+    private AccountType accountType;
+
+    private Location location;
```
Comment (vytaux) [inline src/main/java/com/tg5/service/contract/EventPayload.java:20]: Location in Scanner, not in Event?

File: src/main/java/com/tg5/domain/Event.java [hunk 2]
```diff
@@ -15,14 +15,20 @@ public class Event implements Serializable {
     private static final long serialVersionUID = 1L;
 
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue
```
Comment (vytaux) [inline src/main/java/com/tg5/domain/Event.java]: Let's just keep entities with IDENTITY strategy, prof seems to like it.

File: src/main/java/com/tg5/service/RoleServiceImpl.java [hunk 3]
```diff
@@ -22,7 +22,9 @@ public class RoleServiceImpl extends BaseReadWriteServiceImpl<RolePayload, Role,
 
     @Override
     public RolePayload create(RolePayload request) {
```
Comment (vytaux) [inline src/main/java/com/tg5/service/RoleServiceImpl.java:24]: Are you sure we can't do without this? And if we do create, do we need udpate as well?

File: src/main/java/com/tg5/service/mapper/AccountTypeToAccountTypePayload.java [hunk 4]
```diff
@@ -11,4 +11,11 @@ public class AccountTypeToAccountTypePayload extends BaseMapper<AccountType, Acc
     protected AccountTypeToAccountTypePayload(MapperFactory mapperFactory) {
         super(mapperFactory, AccountType.class, AccountTypePayload.class);
     }
+
+    @Override
```
Comment (vytaux) [inline src/main/java/com/tg5/service/mapper/AccountTypeToAccountTypePayload.java:15]: Is this really necessary?

**Review by vytaux — CHANGES_REQUESTED** [review]

File: src/main/java/com/tg5/service/contract/EventPayload.java (hunk 1 above)
Comment (gombaniro) [reply src/main/java/com/tg5/service/contract/EventPayload.java:20]: The requirement requests that. thinking abroad each event has its location.  At the lecture session teachers, were saying the locations of scanner need to match its preplanned event location

File: src/main/java/com/tg5/domain/Event.java (hunk 2 above)
Comment (gombaniro) [reply src/main/java/com/tg5/domain/Event.java]: For MySQL it does something different, but for postgres,  it seems is equivalent to auto. Do we have to add to all entities

**Review by vytaux — APPROVED** [review]

**Pull request merged**